[core]/cron/server/main.lua

```diff
@@ -12,6 +12,11 @@ local lastTimestamp = false
 ---@param m number
 ---@param cb function|table
 function RunAt(h, m, cb)
+    if type(h) ~= "number" or type(m) ~= "number" or (type(cb) ~= "function" and type(cb) ~= "table") then
```

Comment:
Type validation is already being handled in the `cron:runAt` event. Please remove the redundant code.